react/src/contexts/falcon-api-context.js

```diff
@@ -0,0 +1,22 @@
+import FalconApi from '@crowdstrike/foundry-js';
+import { createContext, useEffect, useMemo, useState } from 'react';
+
+const FalconApiContext = createContext(null);
+
+function useFalconApiContext() {
+  const [isInitialized, setIsInitialized] = useState(false);
+  const falcon = useMemo(() => new FalconApi(), []);
+  const navigation = useMemo(() => falcon.isConnected ? falcon.navigation : undefined, [falcon.isConnected]);
```

Comment:
Why do we need a special case for navigation here? Can't the code in `useCallback` above just call `falcon.navigation.navigateTo()`?